src/components/cards/challenge/Overview.tsx

```diff
@@ -55,6 +55,7 @@ export default function Overview({ challenge, community }: Props) {
             <div className="text-sm md:pl-2 max-w-max">
               <div className="flex gap-1 text-gray-700 font-medium">
                 <span>{shortenNumber(reward.amount)}</span>
+                <span>{reward?.fiatCurrency ? `${reward?.fiatCurrency} in` : ''}</span>
```

Comment:
This is not clear to me, what would be the value displayed if it's a fiatCurrency?,  maybe the naming confuses me

Reply:
Same question, the `in` which is coming after is it the name?

Reply:
the fiatCurrency is the currency in which the organizer of the hackathon gives the funds in.

Reply:
And it is not the name they all add up to be "10K in ICP"